src/components/show-more/show-more.ts

```diff
@@ -20,7 +20,7 @@ export class MShowMore extends ModulVue {
     loading: boolean;
 
     get isVisible(): boolean {
-        return this.nbTotal !== undefined;
+        return this.nbTotal !== undefined && this.nbTotal !== 0;
```

Comment:
On aurait pu faire pour gerer le cas hypothetique du nbTotal negatif

`this.nbTotal !== undefined && this.nbTotal > 0`